packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -143,20 +165,23 @@ class Hyperlink extends React.Component {
     }
 
     return (
-      <ComponentType
-        {...customProps}
-        className={hyperlinkClasses}
-        onKeyDown={this.handleKeyDown}
-        onKeyUp={this.handleKeyUp}
-        onBlur={this.handleOnBlur}
-        onClick={onClick}
-        onFocus={onFocus}
-        href={isDisabled ? null : href}
-        target={target}
-        rel={rel}
-      >
-        {children}
-      </ComponentType>
+      <React.Fragment>
```

Comment:
Is this fragment necessary?

Reply:
👀I don't need it, I have [removed](https://github.com/cerner/terra-core/pull/2666/commits/8f37b997979d8922b14b451f0a9297a1add79b7a) it.